addons/cert-manager/main.tf

```diff
@@ -0,0 +1,17 @@
+module "helm_addon" {
+  source = "../helm"
+
+  manage_via_gitops = var.manage_via_gitops
+  helm_config       = local.helm_config
+  addon_context     = var.addon_context
+
+  depends_on = [kubernetes_namespace_v1.this]
+}
+
+resource "kubernetes_namespace_v1" "this" {
+  count = try(local.helm_config["create_namespace"], true) && local.helm_config["namespace"] != "cert-manager" ? 1 : 0
+
+  metadata {
+    name = local.helm_config["namespace"]
+  }
+}
```

Comment:
resource `kubernetes_namespace_v1` can be removed, as namespace creation is controller by [create_namespace](https://github.com/clouddrove/terraform-aws-eks-addons/blob/0c6885f00e7225355bf16d3dc29c88fc3504831c/addons/cert-manager/locals.tf#L10) variable. Remove Line-8 `depends_on` as well.